src/openttd.cpp

```diff
@@ -603,6 +614,40 @@ int openttd_main(int argc, char *argv[])
 		case 'P':
 			scanner->join_company_password = mgo.opt;
 			break;
+		case 'U': {
+			char* p = mgo.opt;
```

Comment:
We like the '*' to attach to the variable, not the type
```suggestion
			char *p = mgo.opt;
```

Reply:
Oh yes, sorry